### ISSUE TITLE
Strange behaviour in V&V case ver-1e

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [idaholab/TMAP8#73]
I tried running the ver-1e case locally and exported not only the concentration at 15.75 um away from the interface but also at x=20 um (in the first subdomain).

This concentration point shows a high oscillation behaviour (see below). Maybe due to tolerances, mesh or timestep...

![image](https://github.com/idaholab/TMAP8/assets/40028739/16aa3f83-2166-48a9-9b49-4dd457610fc0)

@singhgp4321 any idea?

**Comment by RemDelaporteMathurin** [comment]
This is a paraview animation of the `ver-1e_out.e` file produced by TMAP8

![ezgif-4-5e1bb869d6](https://github.com/idaholab/TMAP8/assets/40028739/98175b07-78da-4828-b850-0cc95fa4833f)

**Comment by singhgp4321** [comment]
Not sure why. Is this with the default parameter values provided in the code or did you change something?

**Comment by RemDelaporteMathurin** [comment]
Yes this is the default parameters, the only thing I changed is the extra export at x = 20 um
I used the [script available on this repo](https://github.com/idaholab/TMAP8/blob/devel/test/tests/ver-1e/ver-1e.i).

**Comment by RemDelaporteMathurin** [comment]
You can only simply open the [exodus file for this test case](https://github.com/idaholab/TMAP8/blob/devel/test/tests/ver-1e/gold/ver-1e_out.e) in Paraview and check for yourself. It shows the same concentration profile as the animation above.

**Comment by singhgp4321** [comment]
Okay, thanks for reporting this. I will get back to you as soon as I have a solution. It may take a week or two as I am tied up with several other tasks right now.

**Comment by ChaitanyaBhave-e** [comment]
The oscillations are entirely caused by too large of an initial time step size, and the use of the Crank-Nicholson integration scheme. This executioner block resolves the fluctuations. I'm opening an issue for this, thanks for pointing out the issue @RemDelaporteMathurin !
```
[Executioner]
  type = Transient
  end_time = 50
  dtmax = 0.2
  solve_type = NEWTON
  petsc_options_iname = '-pc_type -pc_hypre_type'
  petsc_options_value = 'hypre boomeramg'
  scheme = 'crank-nicolson'
  nl_rel_tol = 1e-50 # Make this really tight so that our absolute tolerance criterion is the one
  # # we must meet
  nl_abs_tol = 1e-12
  abort_on_solve_fail = true
  [TimeStepper]
    type = IterationAdaptiveDT
    dt = 1e-4
    optimal_iterations = 4
    growth_factor = 1.25
    cutback_factor = 0.8
  []
[]
```

**Comment by simopier** [comment]
Closed since #115 is merged
